platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -437,14 +439,17 @@ public void fullLeafRehash() {
         try {
             Hash loadedHash = dataSource.loadHash(lastLeafPath);
             if (loadedHash != null) {
-                logger.info("Calculated hash found for the last leaf path: {}, skipping full rehash", lastLeafPath);
+                logger.info(
+                        STARTUP.getMarker(),
+                        "Calculated hash found for the last leaf path: {}, skipping full rehash",
+                        lastLeafPath);
```

Comment:
```suggestion
                logger.info(
                        STARTUP.getMarker(),
                        "Calculated hash found for the last leaf path: {} in the VirtualMap at {}, skipping full rehash",
                        lastLeafPath, getRoute());
```

Reply:
For this log as well as others, I'd suggest logging the merkle route or some other identifying data of the virtual map that is being operated on. If there are a bunch of maps, it's going to be really tricky to figure out which map is being referenced otherwise.